<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnemometerWidget</class>
 <widget class="QWidget" name="AnemometerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget #widgetFrame {
background: rgba(225, 225, 225);
border-radius: 4px;
}

QFrame.containerFrame { 
padding: 3px;
border: solid 15px rgba(175, 175, 175);
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.0 rgba(95, 95, 95),
stop: 0.15 rgba(75, 75, 75),
stop: 0.9 rgba(95, 95, 95),
stop: 1.0 rgba(45, 45, 45));
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 4px;
}

QFrame.containerFrame QLabel {
border: none;
background: transparent;
font-weight: bold;
color: rgba(225, 225, 225);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="widgetFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="class" stdset="0">
         <string>containerFrame</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="topLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="windAngleLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Wind angle</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="windSpeedLabel">
             <property name="text">
              <string>Wind speed</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="background">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>131</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>131</width>
                <height>131</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
background-image: url(./plugins/compassBackground.png);
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <property name="margin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QDial" name="windAngleDial">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>90</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5,
 fy:0.5, stop:0.551136 rgba(7, 114, 255, 0), stop:0.994318 rgba(15, 183, 255, 255), stop:1 rgba(255, 255, 255, 255)); \n color: white;</string>
                 </property>
                 <property name="maximum">
                  <number>359</number>
                 </property>
                 <property name="value">
                  <number>180</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="wrapping">
                  <bool>true</bool>
                 </property>
                 <property name="notchTarget">
                  <double>15.000000000000000</double>
                 </property>
                 <property name="notchesVisible">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSlider" name="windSpeedSlider">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>100</height>
               </size>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="windAngleValueLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>360</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="windSpeedValueLabel">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>0 knots 0.00 m/s</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
